docs/migrate-to-managed-asm/virtual-service.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START servicemesh_migrate_to_managed_asm_virtual_service_virtualservice_frontend_ingress]
+# This VirtualService routes the example.com traffic from my-gateway
+# to port 80 of the hello-world-service.
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: frontend-ingress
+spec:
+  gateways:
+  - frontend-gateway
+  hosts:
+  - '*' # In production, be more specific.
+  http:
+  - route:
+    - destination:
+        port:
+          number: 80
+        host: frontend # The name of the Service inside the cluster you're migrating away from.
+      weight: 50
+    - destination:
+        port:
+          number: 80
+        host: v2.example.com # The temporary domain name of the frontend inside the cluster with managed ASM.
+      weight: 50
+# [END servicemesh_migrate_to_managed_asm_virtual_service_virtualservice_frontend_ingress]
```

Comment:
Note that the folder name is `migration-to-managed-asm`. I have not mentioned `in-cluster` in the folder name or anywhere inside these files because I want to reuse these samples for the [Istio-to-ASM migration tutorial](https://cloud.google.com/service-mesh/docs/migrate-istio-to-anthos-service-mesh).

Reply:
makes sense, thanks for the clarification 👍